pkg/splunk/controller/controller_test.go

```diff
@@ -122,6 +124,14 @@ func (mgr MockManager) Add(r manager.Runnable) error {
 	return mgr.SetFields(r)
 }
 
+func (mgr MockManager) AddMetricsExtraHandler(path string, handler http.Handler) error {
```

Comment:
Was this code auto generated?

Reply:
No this code was not autogenerated, I added these stub functions because we upgraded to "sigs.k8s.io/controller-runtime v0.6.0" per https://sdk.operatorframework.io/docs/upgrading-sdk-version/v0.18.0/

sigs.k8s.io/controller-runtime v0.6.0 has an updated Manager type that includes these functions https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.6.0/pkg/manager#Manager

If we do not include them the unit tests will fail with warnings like 
`pkg/splunk/controller/controller_test.go:117:5: cannot use &MockManager literal (type *MockManager) as type manager.Manager in assignment:
	*MockManager does not implement manager.Manager (missing AddMetricsExtraHandler method)`